### PR TITLE
Feature: Default value for interactive text input

### LINKED ISSUE #576: Default Value for InteractiveTextInput
I would like a default value set for InteractiveTextInput, I have some cases where i would like to prefill the input for easier editing

### PR DESCRIPTION
**Pull request by KarolosLykos**
### Description
Add a default value option for interactive text input.
Also, introduced `WithDefaultValue` method and included additional printer test cases.

### Scope
`InteractiveTextInput`

- [ ] Bug Fix
- [x] New Feature
- [x] Documentation
- [ ] Other

### Related Issue
Fixes #576 

### To-Do Checklist
- [x] I tested my changes
- [x] I have commented every method that I created/changed
- [x] I updated the examples to fit with my changes
- [x] I have added tests for my newly created methods

### CODE REVIEW

File: _examples/interactive_textinput/password/main.go [hunk 1]
```diff
@@ -3,7 +3,11 @@ package main
 import "github.com/pterm/pterm"
 
 func main() {
-	result, _ := pterm.DefaultInteractiveTextInput.WithMask("*").Show("Enter your password")
+	result, _ := pterm.DefaultInteractiveTextInput.
+		//WithDefaultValue("asdasdads").
+		WithMultiLine(true).
+		//WithMask("*").
+		Show("Enter your password")
```
Comment (smitt04) [inline _examples/interactive_textinput/password/main.go:10]: Do we want this example changed? Maybe a new example with default value, as it looks like this should still be a password example. I don't think `WithDefaultValue` really works with `WithMask` either.

File: interactive_textinput_printer.go [hunk 2]
```diff
@@ -226,6 +239,10 @@ func (p InteractiveTextInputPrinter) updateArea(area *cursor.Area) string {
 		}
 	}
 
+	av := internal.GetStringMaxWidth(areaText)
+	vv := internal.GetStringMaxWidth(p.text)
+
+	_ = av + vv
```
Comment (smitt04) [inline interactive_textinput_printer.go:245]: Just curious what this is doing? Does it not update unless you get the max widths?

File: _examples/interactive_textinput/password/main.go (hunk 1 above)
Comment (KarolosLykos) [reply _examples/interactive_textinput/password/main.go:10]: We don't want this example changed, and it was actually left there by accident while I was testing something. 😊

File: interactive_textinput_printer.go (hunk 2 above)
Comment (KarolosLykos) [reply interactive_textinput_printer.go:245]: Oops :D, I was testing something, and these are only debug statements.

**Review by smitt04 — APPROVED** [review]

**Review by MarvinJWendt — APPROVED** [review]
LGTM! Thanks for helping with the review @smitt04 🚀

**Pull request merged**